CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 - Updated deserialization of `bitmap` for custom CPU templates to allow usage
   of '_' as a separator.
+- Upgraded Rust toolchain from 1.66.1 to 1.70.0.
```

Comment:
from 1.68.2, no?

Reply:
The toolchain for the last release `1.3.0` was `1.66.0` so since the last release it changed from `1.66.0` to `1.70.0`.

Reply:
Ah right. Although I guess if it goes into main, then the change will be from 1.4 to 1.5, which would be 1.68.2 to 1.70.0? 

Why are we putting this in the changelog in the first place, though?

Reply:
Removed for simplicity.